scripts/lib.sh

```diff
@@ -158,6 +176,36 @@ check_api_token() {
   fi
 }
 
+check_s3_url() {
+  if [ -z "$EXTENSION_S3_URL" ]; then
+    EXTENSION_S3_URL="https://dynatrace-gcp-extensions.s3.amazonaws.com"
+  else
+    warn "Development mode on: custom S3 url link."
+  fi
+}
+
+check_gcp_config_in_extension() {
```

Comment:
After this PR: https://github.com/dynatrace-oss/dynatrace-gcp-function/pull/209
`check_gcp_config_in_extension` and `check_gcp_config_in_extensions` should be replaced by `validate_gcp_config_in_extensions` function